test/e2e/admission.go

```diff
@@ -51,4 +53,103 @@ var _ = Describe("Job E2E Test: Test Admission service", func() {
 			"Job queue attribute would default to 'default' ")
 	})
 
+	It("Invalid CPU unit", func() {
+
+		context := initTestContext()
+		defer cleanupTestContext(context)
+		namespace := "test"
+
+		var job v1alpha1.Job
+		jsonData := []byte(`{
+   "apiVersion": "batch.volcano.sh/v1alpha1",
+   "kind": "Job",
+   "metadata": {
+      "name": "test-job"
+   },
+   "spec": {
+      "minAvailable": 3,
+      "schedulerName": "volcano",
+      "queue": "default",
+      "tasks": [
+         {
+            "replicas": 3,
+            "name": "default-nginx",
+            "template": {
+               "spec": {
+                  "containers": [
+                     {
+                        "image": "nginx",
+                        "imagePullPolicy": "IfNotPresent",
+                        "name": "nginx",
+                        "resources": {
+                           "requests": {
+                              "cpu": "-1"
+                           }
+                        }
+                     }
+                  ],
+                  "restartPolicy": "Never"
+               }
+            }
+         }
+      ]
+   }
+}`)
```

Comment:
I would suggest moving this to a file.
 and if we want to fix something in the future, it would be easy.